content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -0,0 +1,748 @@
+number at the start of the sequence. This number is often a `1`. Check with the financial
+institution should this first number differ to verify that it is correct, as this number is provided
+for |ACH| approved accounts.
+
+Enter the :guilabel:`Originating DFI Identification` number next, which should contain an assigned
+8-digit number from the financial institution.
+
+.. important::
+   Enter the numerical values in this section *exactly* as the company's financial institution
+   (e.g. bank or credit union) has provided them, otherwise risk failing a successful |NACHA|
+   configuration in Odoo.
+
+.. image:: united_states/us-l10n-nacha-dropdown.png
+   :align: center
+   :alt: NACHA settings with the standard entry class code drop-down menu highlighted.
+
+There are two options for the next field: :guilabel:`Standard Entry Class Code`. Select the
+drop-down menu to the right of the field and pick either :guilabel:`Corporate Credit or Debit (CCD)`
+or :guilabel:`Prearranged Payment and Deposit (PPD)`. Again, this information will be provided by
+the financial institution. By default :guilabel:`Corporate Credit or Debit (CCD)` is selected.
+
+Finally, the last option is for :guilabel:`Generated Balanced Files`. Tick the checkbox to the right
+of the field to enable :guilabel:`Generated Balanced Files`. Consult the company's accountant or
+financial advisor to make an informed decision for this field.
+
+Manually save the configuration by clicking the :icon:`fa-cloud-upload` :guilabel:`(cloud upload)`
+icon, or navigate away from this screen to auto-save. The configuration is now complete.
+
+.. _l10n_us/batch-payment:
+
+Create batch payment
+~~~~~~~~~~~~~~~~~~~~
+
+Now, record each payment in Odoo using the |NACHA| payment method.
+
+.. seealso::
+   :ref:`Register Payments in Odoo <payments/register>`
+
+.. important::
+   Be aware of the cut-off time for same-day payments. Either the file needs to have a future date
+   associated with each payment or the file needs to be sent prior to the cut-off, if the dates
+   included in it match today's date. Consult the financial institution for the exact cut-off time
+   for their processing of same-day payments.
+
+Once all the payments to be included in the |NACHA| |ACH| file have been made, a batch payment needs
+to be made from the :icon:`fa-cog` :guilabel:`Action` menu.
+
+To create the batch payments, access the payments page, by navigating to :menuselection:`Accounting
+--> Vendors --> Payments`. Select all the payments that should be included in the |NACHA| |ACH|
+file, by ticking the checkboxes to the far-left of the rows.
+
+.. image:: united_states/us-l10n-create-batch-payments.png
+   :align: center
+   :alt: On the payments screen, the action menu is highlighted with create a batch payment
+         selected.
+
+.. warning::
+   All payments in the batch must share the same payment method.
```

Comment:
As mentioned above, it's not enough for these to just be the same payment method, they must all be the NACHA payment method.